extensions/spotify-player/src/addCurrentSongToPlaylist.tsx

```diff
@@ -0,0 +1,124 @@
+import { View } from "./components/View";
+import {
+  Action,
+  ActionPanel,
+  Icon,
+  LaunchType,
+  List,
+  Toast,
+  launchCommand,
+  popToRoot,
+  showHUD,
+  showToast,
+} from "@raycast/api";
+import { useState } from "react";
+import { useCurrentlyPlaying } from "./hooks/useCurrentlyPlaying";
+import { useMe } from "./hooks/useMe";
+import { ListOrGridSection } from "./components/ListOrGridSection";
+import PlaylistItem from "./components/PlaylistItem";
+import { addToPlaylist } from "./api/addToPlaylist";
+import { useMyPlaylists } from "./hooks/useMyPlaylists";
+import { getError } from "./helpers/getError";
+
+function AddToPlaylistCommand() {
+  const { currentlyPlayingData, currentlyPlayingIsLoading, currentlyPlayingRevalidate } = useCurrentlyPlaying();
+  const [searchText, setSearchText] = useState("");
+
+  const { myPlaylistsData } = useMyPlaylists();
+  const { meData } = useMe();
+
+  if (!currentlyPlayingData || !currentlyPlayingData.item) {
+    return (
+      <List isLoading={currentlyPlayingIsLoading}>
```

Comment:
If you like I can extract this into it's own component. It's using the same one as in `nowPlaying`